app/views/federations/show.json.jbuilder

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
This is Ruby, so should get the frozen_string_literal magic comment we have in .rb files.